package.json

```diff
@@ -106,6 +106,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^1.7.0",
+    "eslint-config-airbnb-typescript": "^7.2.1",
```

Comment:
don't we need to add `'airbnb-typescript'` to the eslint plugins section to use this?

Reply:
done